src/pages/restaurant/components/menu/components/dishes/components/categoryDishes/components/dish/Dish.js

```diff
@@ -0,0 +1,108 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import dishReader from './readers/dishReader';
+import _noop from 'lodash/noop';
+import getDishIcon from '../../../../../../helpers/getDishIcon';
+
+//Icons
+import rupee from '../../../../../../../../../../images/rupee.svg';
+
+//Components
+import IconWithLabel from '../../../../../../../../../../commonComponents/iconWithLabel';
+import Button from '../../../../../../../../../../commonComponents/button';
+import PlusAndMinusButton from '../../../../../../../../../../commonComponents/plusAndMinusButton';
+import { BUTTON_TYPES } from '../../../../../../../../../../commonComponents/button';
+
+//CSS
+import './dish.css';
+
+function renderDishButton(
```

Comment:
arguments length is more than 3, so we can reduce it to 3 if we wrap all the callback functions inside an object, this object should be passed from the call site